public/css/frdr-theme.css

```diff
@@ -1914,3 +2293,478 @@ a#ndrio_logo {
+    .home-circle {
+        height: 60px;
+        width: 60px;
+    }
+
+    .home-metric-icon {
+        color: var(--alliance-yellow);
+        font-size: 24px !important;
+        justify-content: center;
+        margin-top: 18px;
+    }
+}
+
+/** md size breakpoint **/
+@media (min-width: 769px) and (max-width: 992px) {
+    .home-circle {
+        height: 80px;
+        width: 80px;
+    }
+
+    .home-metric-icon {
+        color: var(--alliance-yellow);
+        font-size: 32px !important;
+        justify-content: center;
+        margin-top: 24px;
+    }
+}
+
+/** lg size breakpoint **/
+@media (min-width: 993px) and (max-width: 1200px) {
+    .home-circle {
+        height: 80px;
+        width: 80px;
+    }
+
+    .home-metric-icon {
+        color: var(--alliance-yellow);
+        font-size: 32px !important;
+        justify-content: center;
+        margin-top: 24px;
+    }
+
+    .home-feature-header {
+        min-height: 5rem;
+    }
+}
+
+/** xl size breakpoint **/
+@media (min-width: 1201px) and (max-width: 1400px) {
+    .home-circle {
+        height: 80px;
+        width: 80px;
+    }
+
+    .home-metric-icon {
+        color: var(--alliance-yellow);
+        font-size: 32px !important;
+        justify-content: center;
+        margin-top: 24px;
+    }
+
+    .home-feature-header {
+        min-height: 5rem;
+    }
+}
+
+/** > xl size breakpoint **/
+@media (min-width: 1400px) {
+    .home-circle {
+        height: 80px;
+        width: 80px;
+    }
+
+    .home-metric-icon {
+        color: var(--alliance-yellow);
+        font-size: 32px !important;
+        justify-content: center;
+        margin-top: 24px;
+    }
+
+    .home-feature-header {
+        min-height: 5rem;
+    }
+}
+
+a.external-link:after {
+    display: inline-block;
+    vertical-align: text-top;
+    font-size: 0.6rem;
+    font-family: "Font Awesome 5 Free";
+    font-weight: 900;
+    content: "\f35d";
+    margin-left: 0.1875rem;
+    margin-right: 0.1875rem;
+    vertical-align: super;
+    color: #282828;
+}
+
+.btn.float {
+    position: fixed;
+    margin-top: calc(80vh);
+    z-index: 1000;
+    margin-left: -2px;
+    padding-top: 10px;
+    padding-bottom: 10px;
+    text-decoration: none;
+}
+
+.home-collection-icon {
+    margin-left: auto;
+    margin-right: auto;
+    height: 80px;
+    width: 80px;
+    object-fit: scale-down;
+}
+
+.carousel-control-prev {
+    background-color: var(--alliance-primary) !important;
+}
+
+.carousel-control-next {
+    background-color: var(--alliance-primary) !important;
+}
+
+/* Alliance Body */
+/* body {
+    margin: 0;
+    font-family: var(--body-font-family);
+    font-size: var(--body-font-size);
+    font-weight: var(--body-font-weight);
+    line-height: var(--body-line-height);
+    color: var(--body-color);
+    background-color: var(--body-bg);
+    -webkit-text-size-adjust: 100%;
+    -webkit-tap-highlight-color: rgba(0,0,0,0);
+} */
+
+
+/* Item List (TEMP) */
```

Comment:
Temp?

Reply:
I can't remember the exact reason of adding this comment. I guess it might be that I copied the code from another css file and planned to come back to see if it works well. I will remove the "Temp" word.